src/utils/cookie.ts

```diff
@@ -0,0 +1,110 @@
+import Cookies from 'js-cookie';
+import { SESSION_COOKIE_EXPIRATION_DURATION_MS } from '../constants/constants';
+
+export interface Session {
+  id: string;
+  token: string;
+  createdAt: number;
+}
+
+export const COOKIE_KEYS = {
+  ACCEPT_COOKIES_KEY: 'accept-all-cookies',
+  SESSION_KEY: 'session',
+  STORED_SESSIONS_KEY: 'storedSessions',
+  REDIRECT_URL_KEY: 'redirectUrl',
+};
+/**
+ * @returns {boolean} whether the user accepted the cookies
+ */
+export const hasAcceptedCookies = () =>
+  Cookies.get(COOKIE_KEYS.ACCEPT_COOKIES_KEY) === 'true';
+
+/**
+ * @param {string|null} token token to set in the session key. A null value will remove it.
+ * @param  {string} domain value for the cookie's domain
+ */
+export const setCurrentSession = (token: string | null, domain: string) => {
+  if (!token) {
+    return Cookies.remove(COOKIE_KEYS.SESSION_KEY, { domain, secure: true });
+  }
+  return Cookies.set(COOKIE_KEYS.SESSION_KEY, token, { domain, secure: true });
+};
+/**
+ * @returns {string|undefined} current session token
+ */
+export const getCurrentSession = () => Cookies.get(COOKIE_KEYS.SESSION_KEY);
+
+/**
+ * @returnss {Session[]} stored sessions
+ */
+export const getStoredSessions = (): Session[] => {
+  try {
+    return JSON.parse(Cookies.get(COOKIE_KEYS.STORED_SESSIONS_KEY) ?? '[]');
+  } catch (e) {
+    return [];
+  }
+};
+
+/**
+ * @param  {Session} session session to store
```

Comment:
The first `session` is the name of the param, then it's the description. But I changed the second session for `value` so it can also read well together!